spyderlib/widgets/arraybuilder.py

```diff
@@ -202,11 +206,11 @@ def __init__(self, parent, inline=True, offset=0):
         if inline:
             self._button_help.setToolTip(self._help_inline)
             self._text = NumpyArrayInline(self)
-            self._widget = self._text
+            self.widget = self._text

```

Comment:
In general I think of private stuff as things only the same object can access, so if another object (even in the same module) can access something, then it is really not private.

So using a thing from within the tests (another place / object) feels like it should be public?